storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
Yes I understood that as the motivation, though it feels ad-hoc somehow to have it in one place but not the other.

FWIW I realize it's almost "impossible" to allow the `Bucket` constructor to support every single settable attribute/property.